app/controllers/licence_finder_controller.rb

```diff
@@ -141,23 +137,28 @@ def extract_ids(param_base)
     ids.sort
   end
 
-  def set_analytics_headers
```

Comment:
I don't think this should have been removed.

The `result_count` will be forwarded to analytics via slimmer to do analysis. Might still be needed.